<launch>
    <node name="nano25_ft"
		pkg="netft_rdt_driver"
		type="netft_node"
		args="10.68.0.120 --rate 10"
		respawn="true">
    <!--remap from="/netft_data" to="/tool_netft/wrench_raw"/-->
    </node>
</launch>
